examples/react/css/src/hits-per-page.css

```
@@ -0,0 +1,30 @@
+.ais-HitsPerPage {
+  &:after {
+    transition: transform var(--ais-transition-duration)
+      var(--ais-transition-timing-function);
+    background-image: var(--ais-chevron-icon);
+    background-repeat: no-repeat;
+    content: '';
+    display: inline-block;
+    height: 1rem;
+    margin-left: calc(calc(var(--ais-spacing) * -1) + -1rem);
```

Could it be simplified to:
```suggestion
    margin-left: calc(var(--ais-spacing) * -1 - 1rem);
```